src/appleseed.studio/mainwindow/project/itembase.cpp

```diff
@@ -49,6 +49,7 @@ ItemBase::ItemBase(
   , m_allow_edition(true)
   , m_allow_deletion(true)
 {
+    setFlags(Qt::ItemIsEnabled | Qt::ItemIsSelectable | Qt::ItemIsUserCheckable);
```

Comment:
Why is this code necessary?

Reply:
I use the QTreeWidgetItem flags to know if I the item can be dragged, but seeing it again make me wonder if a boolean in ItemBase would have been simplier
(this are the default flag without Qt::ItemIsDragEnabled that I had in MaterialItem)

Reply:
My point is, if these are the default flags, why are you setting them?

Reply:
Default flags:  Qt::ItemIsSelectable | Qt::ItemIsUserCheckable | Qt::ItemIsEnabled | Qt::ItemIsDragEnabled | Qt::ItemIsDropEnabled
I remove Qt::ItemIsDragEnabled and Qt::ItemIsDropEnabled for the ItemBase class

Reply:
Ok.

Reply:
So do you prefer a boolean like m_isDragEnabled or we keep using the intern flags of QTreeWidgetItem?

Reply:
I'm fine with the internal flags.